README.md

```diff
@@ -173,6 +173,15 @@ files:
     cleanup_mode: true
 ```
 
+It's also possible to disable escaping special characters (`=`, `:`, `!` and `#`):
+
+```yml
+files:
+  - source: "**/values/strings.xml"
+    translation: "/values-%two_letters_code%/%original_file_name%"
+    escape_special_characters: false
```

Comment:
Can we make it an integer according to the documentation? https://support.crowdin.com/developer/configuration-file/#escape-special-characters

Reply:
fixed